static/src/inline-svgs/journalism/audio-player/fast-forward.svg

```diff
@@ -0,0 +1 @@
+<svg width="26px" height="24px" viewBox="0 0 26 24" xmlns="http://www.w3.org/2000/svg"><title>Fast forward</title><g id="Page-1" fill="none" fill-rule="evenodd"><g id="05-Desktop-HalfWayAudioPage-2" transform="translate(-294 -431)"><g id="Fast-forward" transform="translate(295 431)"><text id="15" font-family="DS5DisplaySansSemibold, DS5 Display Sans Semibold" font-size="15" font-weight="500" fill="#767676"><tspan x="12" y="22.125">15</tspan></text><g id="Arrow"><path d="M11.628701,21.6291694 C16.7636948,21.6291694 20.9264298,17.813329 20.9264298,13.1062514 C20.9264298,8.39917371 16.7636948,4.58333333 11.628701,4.58333333 C8.20537183,4.58333333 4.63794808,4.58333333 0.926429775,4.58333333" id="Oval" stroke="#767676" stroke-width="2" transform="matrix(-1 0 0 1 21.853 0)"/><path id="Triangle" fill="#767676" d="M25 4.58333333L15 9.16666667 15 0z"/></g></g></g></g></svg>
```

Comment:
Hey @ajwl ! This PR is looking good. One thing about these SVGs though, would you be able to follow these docs: https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md? Shout if you have any questions

Reply:
Okay yes can do.